### PR TITLE
Added tests for functions dealing with Caesar and Vigenere ciphers an…

### PR DESCRIPTION
**Pull request by NickLamping**
…d also fixed a few syntax errors

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	empty = ""
+	with open("cae_cipher.txt", 'r') as file:
+		text = file.read().replace('\n','') 
+	with open("plain.txt", 'r') as file:
+		plain = file.read().replace('\n','') 
+
+	assert crack_caeser_cipher(text)[0][2] == plain
+
+def test_vigenere_encrypt():
+	empty = ""
```
Comment (gsingh93) [inline test.py]: Remove

File: test.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	assert vigenere_encrypt(plain, "lemon") == text
+
+def test_get_vigenere_key_length():
+	with open("vig_cipher.txt", 'r') as file:
+		text = file.read().replace('\n','') 
+	assert get_vigenere_key_length(text)[0] == 5 #key is lemon
+
+
+def test_crack_vigenere_cipher():
+	empty = ""
```
Comment (gsingh93) [inline test.py]: Remove

**Comment by NickLamping** [comment]
Remove any tests with file I/O?

Edit: Nevermind, you mean remove those empty strings

**Comment by gsingh93** [comment]
No, I was referring to the unused variable, sorry.

You don't use it anywhere, correct?

**Comment by gsingh93** [comment]
Also, for empty functions, you should use `pass` instead of `empty = ''`.

**Comment by gsingh93** [comment]
One more nit. The rest of our code uses 4 spaces instead of tabs. Any tests should do the same.

**Comment by NickLamping** [comment]
No I was using it so I could denote those functions were to be implemented later. I hadn't heard about `pass` before so thank you for bringing it up.

**Comment by gsingh93** [comment]
Looks good, thanks!

**Pull request merged**